piston/resource.py

```diff
@@ -244,15 +244,14 @@ def _regex_domain_match(self, origin):
                 return origin
 
     def options(self):
-        obj = self._get_obj_or_none()
```

Comment:
nevidím důvod? teoreticky bys měl správně v těch metodách si dávat pozor na to zda tam user je, je to spíš špatně v implementaci těch metod